devices/xiaomi.js

```diff
@@ -1769,4 +1769,24 @@ module.exports = [
             await endpoint1.write('aqaraOpple', {'mode': 1}, {manufacturerCode: 0x115f, disableResponse: true});
         },
     },
+    {
+        zigbeeModel: ['lumi.remote.acn004'],
+        model: 'WXKG17LM',
+        vendor: 'Xiaomi',
+        description: 'Aqara E1 double key wireless switch',
+        meta: {battery: {voltageToPercentage: '3V_2100'}},
+        exposes: [e.battery(), e.battery_voltage(),
+            e.action(['single_left', 'single_right', 'single_both', 'double_left', 'double_right', 'hold_left', 'hold_right']),
+            // eslint-disable-next-line max-len
+            exposes.enum('click_mode', ea.SET, ['fast', 'multi']).withDescription('Click mode, fast: only supports single click which will be send immediately after clicking, multi: supports more events like double and hold'),
```

Comment:
@kirovilya can you check this description?

Reply:
good